src/pip/_internal/req/req_uninstall.py

```diff
@@ -593,7 +593,7 @@ def add(self, entry):
         # backslashes.  This is correct for entries that describe absolute
         # paths outside of site-packages, but all the others use forward
         # slashes.
-        if WINDOWS and not os.path.splitdrive(entry)[0]:
+        if WINDOWS and not os.path.isabs(entry):
```

Comment:
Technically, this will behave differently on a non-absolute path with a drive letter, like `C:foo\bar`. To be honest, though, I can't imagine why we'd ever encounter a real life case where that would happen, so I'm inclined to ignore the issue. And as a general point, `isabs` seems clearer to me.

My only (very slight) reservation is that *not* using `isabs` in the first place is non-obvious enough that thee may have been a reason for it. But it seems to have been introduced in 770930d5550b43f77d5db7543c000500a9641d17, without comment, so I'm inclined to think there was no particular motive.